### ISSUE TITLE
Support for explicit POST method

### ISSUE THREAD

**Issue by bhoomit** [Kvoti/redux-rest#16]
Right now to post something one has to call `actionCreator.create`. 

If I want to use http POST for some other call. I can't. 

In my opinion there should be one custom method parallel to list/retrieve/create/update for such cases.

**Comment by bhoomit** [comment]
What do you think about this feature? Do you think its important? I've added it to my fork. I can send a PR.

**Comment by mallison** [comment]
Can you give the use case for this? It's common to have extra actions that
don't quite fit REST. Eg. /users/:id/set_password -- is that the kind of
thing you mean? Being able to configure such extra actions could be useful.

It's probable the API needs to change to expose the state so that you can
have extra actions and reducers as required. redux-rest's purpose is to
handle the common case but it should be able to integrate with existing
store/actions/reducers.

**Comment by bhoomit** [comment]
Yes, I'm talking about same kind of use cases. Most of my application uses redux-rest, I want to change method for a couple of API calls only. Should I send PR?
